server/routes/handlers/user/userDetails.ts

```diff
@@ -39,12 +39,21 @@ export const postUser = async (req: Request, res: Response): Promise<void> => {
   const processUpload = uploadStorageField('signature')
   processUpload(req, res, async error => {
     try {
-      // let uploadFailed = Boolean(err)
+      if (error) {
+        throw error
+      }
       const { file } = req
+      let signatureBase64
+      if (file) {
+        signatureBase64 = file.buffer.toString('base64')
+      } else {
+        const { signature } = await getUserDetails(uuid, token)
```

Comment:
does this mean we are going back to the server again? dont we already have it loaded on the page (since we are showing it)?

Reply:
true... I guess I could add a hidden field in the form with the value
This is getting quite hacky tho.. after this lets talk about how we develop this